app/views/admin_applications/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render "applications/applicant_info" %>
+<%= render "applications/pet_info", admin: true %>
```

Comment:
I wanted to talk about your use of partials here really quickly. This is an excellent job of making use of partials across directories. In situations where you find yourself using partials across directories, it's more common to find these partials in a `views/shared/` folder. This way your `render`s would look something like this:

```erb
<%= render "shared/applicant_info" %>
<%= render "shared/pet_info", admin: true %>
```